### PR TITLE
Add edit sessions to scenario editor

### PR DESCRIPTION
**Pull request by acang-gs**

### CODE REVIEW

File: stormpiper/stormpiper/spa/src/components/scenario-module/scenario-detail-page.jsx [hunk 1]
```diff
@@ -62,18 +73,54 @@ export default function ScenarioDetailPage({ setDrawerButtonList }) {
...
-        console.log("Found facility: ", res.input.tmnt_facility_collection);
-        setFacility(res.input.tmnt_facility_collection);
-      }
+      buildScenario(res);
+      setCachedScenario(res);
+      setLastCalculatedAt(dateFormatter(res?.result_time_updated));
     });
   }, [params.id, resultsSuccessDisplay]);
 
+  const dateFormatter = (dtValue) => {
```
Comment (austinorr) [inline stormpiper/stormpiper/spa/src/components/scenario-module/scenario-detail-page.jsx:82]: move this to utils, we'll need it for the BMp detail page as well

Comment (acang-gs) [reply stormpiper/stormpiper/spa/src/components/scenario-module/scenario-detail-page.jsx]: moved to utils

File: stormpiper/stormpiper/spa/src/utils/utils.js [hunk 2]
```diff
@@ -53,3 +53,15 @@ export function colorToList(specifier, alpha) {
 export function pick(obj, ...args) {
   return { ...args.reduce((res, key) => ({ ...res, [key]: obj[key] }), {}) };
 }
+
+export function dateFormatter(dtValue) => {
```
Comment (austinorr) [inline stormpiper/stormpiper/spa/src/utils/utils.js:57]: is this line of code valid?

**Review by austinorr — DISMISSED** [review]

File: stormpiper/stormpiper/spa/src/utils/utils.js (hunk 2 above)
Comment (acang-gs) [reply stormpiper/stormpiper/spa/src/utils/utils.js]: oops

**Pull request merged**